gradle/java.gradle

```diff
@@ -10,12 +10,15 @@
  */
 plugins.withId('java') {
     apply plugin: 'checkstyle'
-    apply plugin: 'findbugs'
+    apply plugin: 'com.github.spotbugs'
     apply plugin: 'jacoco'
     apply plugin: 'signing'
     apply plugin: 'maven'
 
     compileJava {
+        sourceCompatibility = "8"
```

Comment:
this is not strictly needed, I can move this in another commit, but it better to have Java version explicitly